addOns/help/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Add Gnome proxy settings user guide
```

Comment:
```suggestion
### Added
- Details for setting Gnome proxy settings.
```

Reply:
Thanks for your example. It is done in the last commit.